src/utils/searchUtils.ts

```diff
@@ -82,7 +82,7 @@ export async function isAntValid(
   }
   const contractTxnData = await arweave.api
     .post('/graphql', buildContractTxQuery(id))
-    .then((res) => {
+    .then((res: any) => {
```

Comment:
nit: we may want to move this function to `SmartweaveContractSource` interface to something like `validateContractSource`

Reply:
not necessary in this PR